### ISSUE TITLE
[PRE REVIEW]: CoreRobotics: An object-oriented C++ library with cross-language wrappers for cross-platform robot control

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#477]
**Submitting author:** @CoreRobotics (<a href="http://orcid.org/0000-0003-1160-0426">Parker Owan</a>)
**Repository:** <a href="https://github.com/CoreRobotics/CoreRobotics" target ="_blank">CoreRobotics/CoreRobotics</a>
**Version:** v0.9.1
**Editor:** @danielskatz
**Reviewer:** @bmagyar

**Author instructions**

Thanks for submitting your paper to JOSS @CoreRobotics. The JOSS editor (shown at the top of this issue) will work with you on this issue to find a reviewer for your submission before creating the main review issue.

@CoreRobotics if you have any suggestions for potential reviewers then please mention them here in this thread. In addition, [this list of people](https://github.com/openjournals/joss/blob/master/docs/reviewers.csv) have already agreed to review for JOSS and may be suitable for this submission.

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
```
PDF failed to compile for issue #477 with the following error: 

 Can't find any papers to compile :-(

```

**Comment by CoreRobotics** [comment]
@whedon commands

**Comment by whedon** [comment]
Here are some things you can ask me to do:

```
# List all of Whedon's capabilities
@whedon commands

# Assign a GitHub user as the reviewer of this submission
@whedon assign @username as reviewer

# List of editor GitHub usernames
@whedon list editors

# List of reviewers together with programming language preferences and domain expertise
@whedon list reviewers

# Change editorial assignment
@whedon assign @username as editor

# Set the software archive DOI at the top of the issue e.g.
@whedon set 10.0000/zenodo.00000 as archive

# Open the review issue
@whedon start review

🚧 🚧 🚧 Experimental Whedon features 🚧 🚧 🚧

# Compile the paper
@whedon generate pdf

```

**Comment by CoreRobotics** [comment]
Hello,

I believe the repository link of the original submission is broken.  The repository is maintained at gitlab, however a mirror is on Github.  The repository for this review needs to be updated to this link: [https://github.com/CoreRobotics/CoreRobotics](https://github.com/CoreRobotics/CoreRobotics)

From the list, @mosteo seems to be an appropriate reviewer.  @jordigh as well.

**Comment by arfon** [comment]
> I believe the repository link of the original submission is broken. The repository is maintained at gitlab, however a mirror is on Github. The repository for this review needs to be updated to this link: https://github.com/CoreRobotics/CoreRobotics

Thanks for spotting this. It's now fixed in this review issue.

> From the list, @mosteo seems to be an appropriate reviewer. @jordigh as well.

Thanks for the suggestions. We'll need an editor to assign themselves first before we can get going with the reviewers.

**Comment by whedon** [comment]
OK, the editor is @danielskatz

**Comment by danielskatz** [comment]
I'll take this, and will start things later today or tomorrow

**Comment by danielskatz** [comment]
@bmagyar - would you be willing to be on the other side of a JOSS paper and do a review?

@mosteo and @jordigh - would one of you be willing to review this as well?

See http://joss.theoj.org/about#reviewer_guidelines for JOSS reviewer guidelines.

**Comment by bmagyar** [comment]
@danielskatz Sure, no problem. Assign it to me

**Comment by danielskatz** [comment]
I will do so, and I will start the review once I get another reviewer, so there's no action for you at this point.

**Comment by danielskatz** [comment]
@whedon assign @bmagyar as reviewer

**Comment by whedon** [comment]
OK, the reviewer is @bmagyar

**Comment by danielskatz** [comment]
@CoreRobotics - Do you have any other reviewer suggestions?  Such as someone else in the robotics community who knows software and is not conflicted with your or your co-authors, but perhaps isn't yet involved with JOSS?  I would like to find a second reviewer.

**Comment by CoreRobotics** [comment]
@danielskatz - Hi Daniel, I'll ask around to see if I can find someone not in conflict.  - Parker

**Comment by Kevin-Mattheus-Moerman** [comment]
@amjaeger17 is this your cup of tea? Are you able to review this?

**Comment by amjaeger17** [comment]
Hi, 
Yeah, I am interested, and can definetly help check off some of the boxes.

**Comment by danielskatz** [comment]
Thanks @amjaeger17 - I'll start the review in a few minutes

**Comment by danielskatz** [comment]
@whedon start review magic-word=bananas

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/489. Feel free to close this issue now!

**Comment by CoreRobotics** [comment]
generate pdf

**Comment by danielskatz** [comment]
@CoreRobotics - this issue is closed - the review is going on in #489

**Comment by CoreRobotics** [comment]
Ok thanks - I didn't realize I could update the pdf in the new thread.
